lms/djangoapps/learner_recommendations/views.py

```diff
@@ -119,6 +141,10 @@ def get(self, request, course_id):
                 if len(recommended_courses) == self.recommendations_count:
                     break
 
+            self._emit_recommendations_viewed_event(
+                request.user.id, is_control, recommended_courses
+            )
+
```

Comment:
Lets move it outside the if condition. We should fire it for people that fall in control group too.